src/main/java/ch/njol/skript/conditions/CondIsInLove.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Animals;
+import org.bukkit.entity.LivingEntity;
+
+@Name("In Love")
+@Description("Whether or not an animal is currently in love")
+@Examples({
+	"on right click on living entity:",
+	"\tif event-entity is in love:",
+	"\t\tsend \"&c&oOhhh, he's in love <3\" to player"
+})
+@Since("INSERT VERSION")
+public class CondIsInLove extends PropertyCondition<LivingEntity> {
+
+	static {
+		register(CondIsInLove.class, "in love", "livingentities");
+	}
+
+	@Override
+	public boolean check(LivingEntity livingEntity) {
+		if(livingEntity instanceof Animals)
```

Comment:
I think we should add a note in description about this, same for other expressions/conditions

Reply:
What would this note be for? As in support animals only?

Reply:
That it only works for animals and not all living entities as the syntax accepts

Reply:
Ah, yeah I'll add a note in description regarding this